addons/point_of_sale/static/src/js/models.js

```diff
@@ -105,6 +105,7 @@ exports.PosModel = Backbone.Model.extend({
 
             return this.connect_to_proxy();
         }
+        return Promise.resolve();
```

Comment:
You should probably addapt the override of 'after_load_server_data' in pos_restaurant as it doesn't return anything. (return the res of super)